spec/controllers/api/v1/book_suggestions_controller_spec.rb

```diff
@@ -0,0 +1,60 @@
+require 'rails_helper'
+
+describe Api::V1::BookSuggestionsController do
+  describe 'POST #create' do
```

Comment:
it is good, but we need to ensure that the record is saved. You can use `change` from rspec to see if the count of the records changed or not

Reply:
Done